api/authentication/auth.py

```diff
@@ -33,22 +36,8 @@ def get_password(username: str):
         :return hashed password from database
     """
     try:
-        db = MySQLdb.connect(host=yc_gc.dbHost, db=yc_gc.dbName, user=yc_gc.dbUser, passwd=yc_gc.dbPass)
-        # prepare a cursor object using cursor() method
-        cursor = db.cursor()
-
-        # execute SQL query using execute() method.
-        results_num = cursor.execute("""SELECT * FROM `users` where Username=%s""", (username, ))
-        db.close()
-        if results_num == 0:
-            return None
-        else:
-            data = cursor.fetchone()
-            return data[2]
-
-    except MySQLdb.MySQLError as err:
-        if err.args[0] not in [1049, 2013]:
-            db.close()
+        return User.query.filter_by(Username=username).first().Password
+    except MySQLError as err:
```

Comment:
I am not sure if this error is relevant here at all. will mysqlerror be ever thrown? there is SQLAlchemyError base exception for all the sqlalchemy errors if you are aiming to catch them which for me seems like the case in here